kolibri/core/content/tasks.py

```diff
@@ -67,7 +67,7 @@ def get_status(job):
 
 class ChannelValidator(JobValidator):
     channel_id = HexOnlyUUIDField()
-    channel_name = serializers.CharField()
+    channel_name = serializers.CharField(default="", allow_blank=True)
```

Comment:
For most of the cases (different forms of import, mainly) I think this is fine, but I see `DeleteChannelValidator` uses a class that inherits from `ChannelValidator`. I think that, as a safety measure, it's good to have both the `id` and the `name` to confirm the deletion of a channel.